src/solve.jl

```diff
@@ -296,8 +296,35 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
 
     # set up additional initial values of newly created DDE integrator
     # (such as fsalfirst) and its callbacks
-    initialize!(dde_int)
-    initialize!(integrator.opts.callback, integrator.t, u, dde_int)
+
+    if initialize_integrator
```

Comment:
According to the failed tests it has to be added to list of keywords, as in OrdinaryDiffEq.